shibata revised for #37, 38, 39,40 (#42)
fujitake/vantiq-related: vantiq-apps-development/1-day-workshop/docs/eng/2-Lab01_Preparation.md

## ***Step 2***

1. When "New Project" Wizard will be displayed, select "New Vantiq Namespace" and set the Namespace name for **Data Generator** to Name. Click _Continue_ button.  
<img src="../../imgs/Lab01/image0.png" width=70%>     

#### Reference
  - [2.3: Organizing Resources](https://community.vantiq.com/courses/developer-level-1-course/lessons/2-the-vantiq-platform/topic/2-3-organizing-resources-master/)

## ***Step 3（Import the Data Generator）***

> ＊ Prepare the Data Generator to generate mock data which is simulate the data from the temperature sensors and the RPM sensors attached to multiple pumps respectively.

Reviewer: ```suggestion
＊ Prepare the Data Generator to generate mock data that simulate the temperature sensors and the RPM sensors attached to each of multiple pumps.
```

4. Click the _Edit_ (a small pencil) icon to open "Edit Server URI" dialog.

    <img src="../../imgs/Lab01/image02.png" width=60%>  

> 5. "Server URI:" is set to a temporary value in advance, so reconfigure it to the URI of the MQTT Broker server which you prepared by yourself. Click the _OK_ button.

Reviewer: ```suggestion
5. "Server URI:" has preset  temporary value, so you need to reconfigure it to that of the MQTT Broker server which you prepared by yourself. Click the _OK_ button.
```

    TempSensor Topic： _/***your name***/pump/Temp_

>     ＊ In the _your name_ field, type your name or any other value that is **not identical to others**. Also, the Topic name should **not contain double quotation marks**. Please do **not include spaces before or after** the Topic name.  

Reviewer: ```suggestion
    ＊ In the _your name_ field, type your name **to make the topic name unique**. Also, make sure that the Topic name should **NOT** have enclosing extra spaces nor double quotations.  
```

        ✔︎   **Temp** SensorTopic：　/***your name***/pump/**Temp**

> -   Whether the project that is opened is correct.

Reviewer: ```suggestion
-   Whether you have correct Namespaces and Projects.
```

>     -   There will be two development Namespaces. One Namespace has the `TrainingDataGen` Project, which allows to review the Resources used by the Data Generator, and another Namespace has the `PumpFailureDetection` Project, which adds the Resources used to build the **Pump Failure Detection System** (currently empty). Work on the `PumpFailureDetection` Project from the next Lab.

Reviewer: ```suggestion
    -   There will be two development Namespaces;  One should have the `TrainingDataGen` Project which contains those Resources used by the Data Generator, and the other should have the `PumpFailureDetection` Project to which Resources to build the **Pump Failure Detection System** (currently empty) will be added.  We will work on the `PumpFailureDetection` Project from the next Lab.
```
